e2e-test/workspace_helper.py

```diff
@@ -61,6 +61,63 @@ def create_workspace(billing_project_name, azure_token, rawls_url, workspace_nam
     else:
         raise Exception(f"Error creating workspace: retries maxed out.")
 
+
+# CREATE GCP WORKSPACE ACTION
+def create_gcp_workspace(billing_project_name, token, rawls_url, workspace_name = "", auth_domains = [], enhanced_bucket_logging = False):
+    for i in range(0,shared_variables.RETRIES):
+        try:
+            # create a new workspace, need to have attributes or api call doesnt work
+            rawls_workspace_api = f"{rawls_url}/api/workspaces"
+            workspace_name = workspace_name if workspace_name else f"e2e-test-api-workspace-{''.join(random.choices(string.ascii_lowercase, k=5))}"
+            logging.info(f"Creating workspace {workspace_name} in {billing_project_name}")
+            
+            request_body = {
+                "namespace": billing_project_name,
+                "name": workspace_name,
+                "authorizationDomain": [{"membersGroupName": auth_domain} for auth_domain in auth_domains],
+                "enhancedBucketLogging": enhanced_bucket_logging,
+                "attributes": {}
+            }
+            header = {
+                "Authorization": "Bearer " + token,
+                "accept": "application/json"
+            }
+
+            workspace_response = requests.post(url=rawls_workspace_api, json=request_body, headers=header)
+            if workspace_response.status_code != 201:
```

Comment:
interesting i thought this would be a `202` like the other saga APIs

Reply:
for GCP it's not an async transaction. see server response section: https://rawls.dsde-prod.broadinstitute.org/#/workspaces/createWorkspace

Reply:
ahhh interesting, fair